vm/chef/cookbooks/logstash/files/syslog.conf.template

```diff
@@ -0,0 +1,16 @@
+input {
```

Comment:
do we need this file?
looks like it's not copied to the image and would be replaced here anyway
https://github.com/GoogleCloudPlatform/click-to-deploy/pull/1059/files#diff-42b3399e6810399bb7be9b989336f68b3f716ce1b2b5b488c66cc5b1c490e1b3R52

Reply:
File removed.